geode-core/src/main/java/org/apache/geode/internal/cache/PRQueryProcessor.java

```diff
@@ -203,117 +185,10 @@ private void executeWithThreadPool(Collection<Collection> resultCollector)
     if (reattemptNeeded) {
       throw fre;
     }
-
-  }
-
-  /**
-   * @throws ForceReattemptException if bucket was moved so caller should try query again
-   */
-  private void doBucketQuery(final Integer bId, final PartitionedRegionDataStore prds,
```

Comment:
@jhuynh1 This private method was not being called anywhere.  Is it safe to remove this dead code?